README.md

```diff
@@ -104,6 +104,7 @@ The purpose of this document is to provide a quick overview over existing packag
 - [steam-api](https://github.com/DaMitchell/steam-api-php) - A PHP wrapper for the Steam API.
 - [steamid](https://github.com/DoctorMcKay/php-steamid) - SteamID class for PHP.
 - [steam-totp](https://github.com/DoctorMcKay/php-steam-totp) - PHP library to deal with Steam's proprietary TOTP algorithm.
+- [steam-auth](https://github.com/vikas5914/steam-auth) - Alternative PHP Steam Authentication with Composer Support
```

Comment:
Use normal sentence capitalization, not title capitalization. Also add a period. Maybe something like:

> An alternative Steam authentication library with Composer support.